computable/contracts/voting.py

```diff
@@ -4,6 +4,26 @@ class Voting(Deployed):
     def at(self, w3, address):
         super().at(w3, address, 'voting.abi')
 
+    def set_privileged(self, parameterizer, datatrust, listing, investing, opts=None):
```

Comment:
Should we expose set_privileged? In the final phase of hardening, we'll lock it down so it can't be called more than once, so does it need an API method? Maybe needed for the initial deployment though...

Reply:
I had the same thought - that i would not. but without it no specs can ever be written